### ISSUE TITLE
DDS files compressed with compressonator failing to load in game [Valkyria Chronicles]

### ISSUE THREAD

**Issue by NikoofDeath** [GPUOpen-Tools/compressonator#149]
I'm trying to use compressonator to decompress, upscale, and recompress textures for Valkyria Chronicles. However, strangely, the textures compressed with compressonator appear to be broken in some manner, as they cause an infinite loading screen in the game, while textures compressed using GIMP's DDS plugin work fine. I used `CompressonatorCLI.exe -EncodeWith HPC -Quality 1.0 -fd DXT1` to compress the file (additionally, I tried EncodeWith CPU, fd BC1, and various mipmap settings). [Here is a zip of the PNG file I'm compressing, the (working) GIMP compressed DDS file, and the (not working) compressonator compressed DDS file.](https://github.com/GPUOpen-Tools/compressonator/files/5469760/valwRF01.zip) Looking at the two files info I cant find any differences that would be causing this problem. If there's anything I can do to provide more information let me know and I'll try to contribute it.
Perhaps related, some of the textures from the game fail to decompress (there was an additional error I was getting for other files when using vulkan to decompress, but using CPU (or whatever the default is) fixes those ones). An example message:
```
Error [3]: DDS Plugin Failed to load texture file C:\Compressonator_4.0.4855\bin\CLI\dds\resource\mx\mmf\evmap09_05\23.dds
Error: loading source image
```
[Here is the DDS file in question.](https://github.com/GPUOpen-Tools/compressonator/files/5469770/23.zip)

**Comment by NPCompress** [comment]
@NikoofDeath , The file not working compressed 00compressonator.dds file,  is not a compressed dds file. Its the same as the sample 00.png, can you check that you attached a compressed version. Thx

**Comment by NikoofDeath** [comment]
> @NikoofDeath , The file not working compressed 00compressonator.dds file, is not a compressed dds file. Its the same as the sample 00.png, can you check that you attached a compressed version. Thx

What makes you say that its not compressed? It shows up in xnview as having BC1 compression, and when compared visually to the PNG there are compression artifacts visible, and GIMP is able to load up its mipmaps. Additionally, it is the same size as the GIMP BC1 compressed file.

**Comment by NPCompress** [comment]
@NikoofDeath. Strange I used Visual Studio 2017 to view the file properties as a second confirmation and its shows up as 32bpp RGBA. Just tried another app DDSView.exe that uses DirectX (other then Compressonator which does show it as compressed bc1). and the file is viewable as compressed image.

![image](https://user-images.githubusercontent.com/18406717/97790457-16ba7300-1b9f-11eb-81dc-93623ba988a9.png)

![image](https://user-images.githubusercontent.com/18406717/97790475-436e8a80-1b9f-11eb-8954-bb8782b11b0c.png)

So the 00Compressonator.dds file appears to be ok!

Also tried this command line option in the same folder as cli application and it processed correctly!.
![image](https://user-images.githubusercontent.com/18406717/97790517-86c8f900-1b9f-11eb-816f-893fadc8f8c1.png)

Will need to look closer at the GIMP sample dds file format for differences.

**Comment by NikoofDeath** [comment]
[If it's any help, here's the original file from the game.](https://github.com/GPUOpen-Tools/compressonator/files/5469911/00.zip)
The steps I took were extracting this from the game files, decompressing to PNG using Compressonator, upscaling using ESRGAN, then compressing the upscaled PNG back to dds using compressonator/gimp, and injecting the new texture into the game files, with the compressonator file not working in game, and the gimp file working in game.

**Comment by NPCompress** [comment]
@NikoofDeath  Does this sample load in the game?

[00_PNG_BC1_1.zip](https://github.com/GPUOpen-Tools/compressonator/files/5549124/00_PNG_BC1_1.zip)

**Comment by NikoofDeath** [comment]
@NPCompress Yup, seems to be working perfectly!
Also curious how you further compressed it below 340kb while still maintaining the same resolution and alpha channel.

**Comment by NPCompress** [comment]
@NikoofDeath That's good. This is a sample generated from our v4.1 release scheduled for next week.

There has been many changes to the code both in bug fixes including: sdk lib changes, updates to dds, ktx file io, smaller installation files, cmake restructuring and more.

**Comment by NPCompress** [comment]
@NikoofDeath Is this issue resolved for you using Compressonator V4.1.5083?

**Comment by NPCompress** [comment]
The issue has been resolved in current release v4.4
